### ISSUE TITLE
Fix Ambient sound volumes

### ISSUE THREAD

**Issue by JJChalupnik** [DamnedNForsaken/MMLX#165]
The music and ambient noise from inside the dungeon, needs to fade off completely as Megaman goes up the elevator. 

May require just re-setting up the music cues on a different player object. But this is something that will greatly help the atmosphere.

**Comment by JJChalupnik** [comment]
This is working but I think we need to fix the boss music so it fades out when the Hanmuru dies. 

In context this may not matter, because we will be doing cutscenes and fighting the Hanmuru on the tower exterior so 🤷‍♂️

**Comment by DamnedNForsaken** [comment]
Yeah i think this is still an issue i have tried to kill all sound after the love music but to no avail. This is still outstanding.

**Comment by DamnedNForsaken** [comment]
My last comment was fixed in one of the recent PRs, with the refactor of the credits
